src/parser/core/CASTS_THAT_ARENT_CASTS.ts

```diff
@@ -58,6 +58,10 @@ const spells: number[] = [
   SPELLS.COMBAT_MEDITATION_TRIGGER.id, //The Spell ID when Combat Meditation from Kyrian Soulbind triggers, so that there is now a soul ready to pick up
   SPELLS.COMBAT_MEDITATION_PICK_UP.id, //The Spell ID that is "cast" when an orb is picked up
   //endregion
+
+  //region Demon Hunter
+  SPELLS.FELBLADE_PAIN_GENERATION.id, //The spell that generates fury when casting 
+  //endregion
 ];
```

Comment:
Spell should be renamed to match it generating fury and not pain

Reply:
True. I noticed its both fury gen + damage. Would FELBALDE_DAMAGE be accepted in such case?

Reply:
Sure - with correct felblade spelling 😉

Reply:
Haha its correct in the code 💯